src/compiler/scala/tools/nsc/util/ThreadIdentityAwareThreadLocal.scala

```diff
@@ -0,0 +1,39 @@
+package scala.tools.nsc.util
+
+object ThreadIdentityAwareThreadLocal {
```

Comment:
Maybe a object Parallel, with all of the Parallel compilation extras in one place?